packages/fx-core/src/plugins/resource/bot/appStudio/appStudio.ts

```diff
@@ -152,6 +152,33 @@ export class AppStudio {
     return app;
   }
 
+  public static async getBotRegistration(
+    accessToken: string,
+    botId: string
+  ): Promise<IBotRegistration | undefined> {
+    const axiosInstance = AppStudio.newAxiosInstance(accessToken);
```

Comment:
The axiosInstance is also created in L158, is there a better way to avoid 2 axios instances?